### DISCUSSION TITLE
Nestjs why not work { emit: 'event', level: 'info' },

### DISCUSSION THREAD

**Discussion by n4an** [prisma/prisma#10323] [Q&A]
why not work `{ emit: 'event', level: 'info' }`,
but work `{ emit: 'event', level: 'query' }`,

```ts
this.$on<any>('query', (event: any) => {
      this.logger.log('Executing query: ' + event.query +' ( ' + event.duration + 'ms )')
})
this.$on<any>('info', (event: any) => {
      this.logger.log('Info: ' + event.message + ' | ' + event.timestamp)
})
```

**Comment by austincrim** [comment]
Hi @Mitch-i 👋,

When you say that the `info` level logs don't work, are you receiving an error or are they simply not showing up?
Would you mind sharing a minimal reproduction repo so I can investigate this?

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
